Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/HOKSAMLAssertionBuilder.java

```diff
@@ -380,35 +355,23 @@ protected Subject createSubject(final String x509Name, final X509Certificate cer
     public Evidence createEvidence(final CallbackProperties properties, final Subject subject) {
         LOG.debug("SamlCallbackHandler.createEvidence() -- Begin");
         final String evAssertionID = properties.getEvidenceID();
-        final DateTime issueInstant = properties.getEvidenceInstant();
-        final String format = properties.getEvidenceIssuerFormat();
-        final DateTime beginValidTime = properties.getEvidenceConditionNotBefore();
-        final DateTime endValidTime = properties.getEvidenceConditionNotAfter();
         final List<AttributeStatement> statements = createEvidenceStatements(properties);
-
-        final String issuer = properties.getEvidenceIssuer();
-
-        return buildEvidence(evAssertionID, issueInstant, format, beginValidTime, endValidTime, issuer, statements,
-                subject);
+        return wrapperEvidence(new WrapperEvidenceParameter(evAssertionID, properties, statements, subject));
```

Comment:
Two stylistic suggestions to consider here:

1. Changing to "wrapEvidence" would follow the normal verb-/action-based method naming convention.
2. "Wrapper" in a class name would usually go last, not first; e.g., org.apache.wss4j.common.saml.**SamlAssertionWrapper**